src/modules/dashboard/components/SubscribedColoniesList/SubscribedColoniesDropdown.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import ColonyAvatar from '~core/ColonyAvatar';
+import DropdownMenu, {
+  DropdownMenuSection,
+  DropdownMenuItem,
+} from '~core/DropdownMenu';
+import Popover from '~core/Popover';
+import { ProcessedColony } from '~data/generated';
+import styles from './SubscribedColoniesList.css';
+
+const displayName =
+  'dashboard.SubscribedColoniesList.SubscribedColoniesDropdown';
+
+export type Colony = Pick<
+  ProcessedColony,
+  | 'colonyName'
+  | 'colonyAddress'
+  | 'id'
+  | 'displayName'
+  | 'avatarHash'
+  | 'avatarURL'
+>;
+interface Props {
+  activeColony: Colony;
+  coloniesList: Colony[];
+}
+
+const SubscribedColoniesDropdown = ({ activeColony, coloniesList }: Props) => {
+  return (
+    <Popover
+      content={() => {
+        return (
+          <DropdownMenu>
+            <DropdownMenuSection>
+              {coloniesList.map((colony) => (
+                <DropdownMenuItem key={colony.colonyAddress}>
+                  <NavLink
+                    activeClassName={styles.activeColony}
+                    title={colony.colonyName}
+                    to={`/colony/${colony.colonyName}`}
+                  >
+                    <div className={styles.dropdownItem}>
+                      <div className={styles.itemImage}>
+                        <ColonyAvatar
+                          colony={colony}
+                          colonyAddress={colony.colonyAddress}
+                          size="xs"
+                        />
+                      </div>
+                      <div>{colony.displayName}</div>
+                    </div>
+                  </NavLink>
+                </DropdownMenuItem>
+              ))}
+            </DropdownMenuSection>
+          </DropdownMenu>
+        );
+      }}
+      trigger="click"
+      showArrow={false}
+      popperOptions={{
+        modifiers: [
+          {
+            name: 'offset',
+            options: {
+              offset: [0, 5],
+            },
+          },
+        ],
+      }}
+    >
+      <NavLink
+        activeClassName={styles.activeColony}
+        className={styles.itemLink}
+        title={activeColony.colonyName}
+        to={`/colony/${activeColony.colonyName}`}
+      >
+        <div className={styles.itemImage}>
+          <ColonyAvatar
+            colony={activeColony}
+            colonyAddress={activeColony.colonyAddress}
+            size="xs"
+          />
+        </div>
+      </NavLink>
```

Comment:
I wanted to extract this into a `SubscribedColony` component, as it's also used in the desktop render of the `SubscribedColoniesList`. However, extracting it into its own component and passing that component as the child of Popover broke the popover (dropdown menu didn't open on click). Would need to spend an amount of time disporportionate to the benefit gained to try to figure out why (tried a little bit but to no avail), but note here in case its actually an easy fix.

Reply:
Try wrapping the `Popover` in a `div` before rendering it

I _think_ that might solve your issue

Reply:
I played around with it a little bit, putting `div`s inside the component, around the component, passing through the ref provided by the `Popover` element, using forward ref... It seems to be specifically due to this `Nav Link` block because rending anything else inside the `Popover` works. Puzzling.